### ISSUE TITLE
'GO:0016358 dendrite development' is missing a parent

### ISSUE THREAD

**Issue by BarbaraCzub** [geneontology/go-ontology#15417]
'GO:0016358 dendrite development' is not related to 'GO:0031175 neuron projection development'.

Should it be a direct is_a child of 'GO:0031175 neuron projection development'?

Or should it be a part_of child of '[POTENTIAL NEW TERM] dendritic tree development'? (compare the relationships in the cellular_component branch - screenshot below) 

And then should this '[POTENTIAL NEW TERM] dendritic tree development' become an is_a child of 'GO:0031175 neuron projection development'?

<img width="706" alt="screen shot 2018-03-14 at 17 35 05" src="https://user-images.githubusercontent.com/16541096/37420494-d48807ae-27ae-11e8-97f3-bea4a344dbfd.png">

@paolaroncaglia @ukemi could you please advise?

cc @RLovering

**Comment by paolaroncaglia** [comment]
@BarbaraCzub 
Apologies, for a number of reasons I could not address this ticket so far, and won't be able to until (mid) next week. At a quick glance: I can't remember if the editors resolved that 'development of x', where x is part_of y, should be is_a 'development of y' rather than a part_of it. If so, the reasoner should infer it and I'm not sure why this isn't happening. 
@ukemi could you advice please? Many thanks.

**Comment by ukemi** [comment]
@paolaroncaglia , you are correct. These need to be handled manually and if anatomical structure A is part of anatomical structure B, then anatomical structure A development should be part of anatomical structure B development.

**Comment by paolaroncaglia** [comment]
Thanks @ukemi .
Summing up, @BarbaraCzub , GO:0031175 'neuron projection development' should be part_of parent of GO:0016358 'dendrite development', or of GO:NEW 'dendritic tree development', depending on https://github.com/geneontology/go-ontology/issues/15412.

**Comment by BarbaraCzub** [comment]
Thank you @ukemi for confirming I was supposed to add this relationship manually.

@paolaroncaglia from David's comment it seems to me that, if we are making GO:0031175 'neuron projection development' the parent term of GO:0016358 'dendrite development', then the relationship between these terms should be is_a (and not part_of), because the relationship between  'neuron projection' and 'dendrite is is_a. So this is what I'll do (instead of adding GO:NEW 'dendritic tree development').

But now I do not know whether I can still merge this pull request (https://github.com/geneontology/go-ontology/pull/15413), if I created it several days ago? (The Travis checks pass every time, when I open this webpage).
Or has the ontology changed too much since I opened it, so things will crash, if I try to merge now, and so I should close it without merging, and delete it? And then re-do this edit using the current ontology version?
